benchmarks/styled-components/webpack.config.js

```diff
@@ -11,5 +25,9 @@ module.exports = {
       },
     ],
   },
-  plugins: [new HtmlWebpackPlugin()],
+  plugins: [new HtmlWebpackPlugin({
+    template: `${__dirname}/src/index.html`,
```

Comment:
We don't need a template, just a title.

Reply:
I'll remove the unnecessary `webpack.config.js` declarations.